packages/ckeditor5-widget/tests/widgetresize/sizeview.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+import SizeView from '../../src/widgetresize/sizeview';
+import ResizerState from '../../src/widgetresize/resizerstate';
+
+describe( 'SizeView', () => {
```

Comment:
If we're testing this view properly now, then some essential tests are missing:

*   template structure assertion,
*   `style.display` tests,
*   children text (label) tests.